update.mjs

```diff
@@ -477,7 +477,15 @@ const svelteKitRouting = {
 }
 
 function stringify(items) {
-  return Array.from(new Set(items)).sort().join(', ')
+  return Array.from(new Set(
+    /**
+     * Exclude items with more than one wildcard.
+     *
+     * See: [feat(app): always exclude multiple wildcards](https://github.com/antfu/vscode-file-nesting-config/pull/245)
+     */
+    items.filter(item => item.split('*').length <= 2),
```

Comment:
Might be better to just throw an error and let the script crash to not update the generated config

Reply:
Sounds great! Updated!